services/venice-server/src/main/java/com/linkedin/venice/listener/ListenerService.java

```diff
@@ -102,7 +102,6 @@ public ListenerService(
           serverConfig.getSslHandshakeThreadPoolSize(),
           "SSLHandShakeThread",
           serverConfig.getSslHandshakeQueueCapacity());
-      new ThreadPoolStats(metricsRepository, this.sslHandshakeExecutor, "ssl_handshake_thread_pool");
```

Comment:
Why this change?

Reply:
Also Server is not using the dns offload logic, and I wondered whether it is necessary or not..
Maybe it doesn't since Server is using H2.

Reply:
It's not DNS offload, but SSL offload. Server is doing SSL offload properly. You could double check "HttpChannelInitializer#initChannel"

Reply:
Got it, but why removing the stats?

Reply:
It's now initialized in "HttpChannelInitializer"; otherwise, I need to pass the stats from ListenerService into HttpChannelInitializer.